syntax_and_semantics/assignment.md

```diff
@@ -62,3 +62,88 @@ objects[1] += 2        # same as: objects[1] = objects[1] + 2
 objects[1] ||= 2       # same as: objects[1]? || (objects[1] = 2)
 objects[1] &&= 2       # same as: objects[1]? && (objects[1] = 2)
 ```
+
+# Chained assignment
+
+You can assign multiple variables to one value at the same time using the chained assignment:
+
+```crystal
+var1 = var2 = var3 = 123
+
+# The above is the same as this:
```

Comment:
It's actually not the same and it's only equivalent in some cases.
The value is duplicated and not evaluated multiple times. For example with `a = b = c = rand` all variables have the same value.
I would only add the resulting values to the example:
```cr
a # => 123
b # => 123
c # => 123
```

And maybe there could be a note that this works because the assignment operator returns the assigned value and a chained assignment expression is essentially `a = (b = (c = 123))`.